DESCRIPTION

```diff
@@ -31,7 +32,16 @@ URL: http://rmarkdown.rstudio.com/flexdashboard
 BugReports: https://github.com/rstudio/flexdashboard/issues
 Encoding: UTF-8
 Depends: R (>= 3.0.2)
-Imports: tools, jsonlite, htmltools, knitr (>= 1.13), htmlwidgets (>= 0.6), rmarkdown (>= 1.10.3), shiny (>= 0.13)
+Imports: 
+  tools, 
+  jsonlite, 
+  htmltools, 
+  knitr (>= 1.13), 
+  htmlwidgets (>= 0.6), 
+  rmarkdown (>= 2.2), 
+  shiny (>= 0.13), 
+  scales, 
+  bslib
```

Comment:
Are we sure we want to take a hard dependency on bslib? If we're going to do other things that need a hard dependency, it makes sense, but if it's just for gauges, that's less compelling.

Reply:
Yea, I decided to add it now since I think it's going to make the future theming PRs easier

We can revisit whether it's worth it or not again